frontend/src/components/CardCreator/CardCreator.tsx

```diff
@@ -1,131 +1,444 @@
+import earth from "../../Media/Attributes/EARTH.png";
+import fire from "../../Media/Attributes/FIRE.png";
+import water from "../../Media/Attributes/WATER.png";
+import wind from "../../Media/Attributes/WIND.png";
+
+
+import levelstar1 from "../../Media/Level/levelstar.png";
 
 function CardCreator() {
+  const [showedStars, setShowedStars] = useState(
+    <div className="star-container"></div>
+  );
+
+  const [showedRace, setShowedRace] = useState(<div></div>);
+  const [showedAttribute, setShowedAttribute] = useState(
+    <img src={dark} alt="" />
+  );
+  const [showedType, setShowedType] = useState(<div></div>);
+  const [showedCard, setShowedCard] = useState(<div className="card-container"><img className="card_img" src={effectCard} alt="" /></div>);
+  const [name, setName] = useState('');
+  const [description, setDescription] = useState('')
+  const [attack, setAttack] = useState("");
+  const [defence, setDefence] = useState("");
+
+
+  const changeDark = () => {
+    setShowedAttribute(<img src={dark} alt="" />);
+  };
+
+  const changeLight = () => {
+    setShowedAttribute(<img src={light} alt="" />);
+  };
+
+  const changeDivine = () => {
+    setShowedAttribute(<img src={divine} alt="" />);
+  };
+
+  const changeEarth = () => {
+    setShowedAttribute(<img src={earth} alt="" />);
+  };
+
+  const changeFire = () => {
+    setShowedAttribute(<img src={fire} alt="" />);
+  };
+
+  const changeWater = () => {
+    setShowedAttribute(<img src={water} alt="" />);
+  };
+
+  const changeWind = () => {
+    setShowedAttribute(<img src={wind} alt="" />);
+  };
+
+
+  const changeType = (types: string) => {
+    if (types === "Effect") {
+      setShowedType(<div>Effect</div>);
+      setShowedCard(<div className="card-container"><img className="card_img" src={effectCard} alt="" /></div>)
+    } else if (types === "Normal") {
+      setShowedType(<div>Normal</div>);
+      setShowedCard(<div className="card-container"><img className="card_img" src={normalCard} alt="" /></div>)
+    } else if (types === "Ritual") {
+      setShowedType(<div>Ritual</div>);
+      setShowedCard(<div className="card-container"><img className="card_img" src={ritualCard} alt="" /></div>)
+    } else if (types === "Synchro") {
+      setShowedType(<div>Synchro</div>);
+      setShowedCard(<div className="card-container"><img className="card_img" src={synchroCard} alt="" /></div>)
+    } else if (types === "Xyz") {
+      setShowedType(<div>Xyz</div>);
+      setShowedCard(<div className="card-container"><img className="card_img" src={xyzCard} alt="" /></div>)
+    } else if (types === "Fusion") {
+      setShowedType(<div>Fusion</div>);
+      setShowedCard(<div className="card-container"><img className="card_img" src={fusionCard} alt="" /></div>)
+    } else if (types === "") {
+      setShowedType(<div></div>);
+      
+    }
+  };
+
+  const changeRace = (races: string) => {
+    if (races === "Aqua") {
```

Comment:
This method can be written in one line. Just use the parameter `races` inside the div and you won't need a single if.

Reply:
I managed to make that work as you said.